### DISCUSSION TITLE
如何获取群里发送文件的消息

### DISCUSSION THREAD

**Discussion by police0909** [Mrs4s/go-cqhttp#2008] [Q&A]
发送群文件，好像没有上报事件啊，怎么获取别人上传群文件的动态啊

**Comment by yyx2684264** [comment]
其他人上传完成群文件后应该是会收到消息的：
`群 机器人测试交流群群(12341234) 内 摸凹猫(4321234) 上传了文件: 你们不要再打了啦_.gif`

后台程序应该会收到如下内容的事件信息：
`{
  post_type: 'notice',
  notice_type: 'group_upload',
  time: 1683596743,
  self_id: 1075879478,
  group_id: 12341234,
  user_id: 4321234,
  file: {
    busid: 102,
    id: '/75cc13a2-3d3d-4eaa-8710-9d4809f43aab',
    name: '你们不要再打了啦_.gif',
    size: 1718648,
    url: 'http://113.249.150.166/ftn_handler/9e0386eeff2053f376e4c96352f2d76b075debc0a0a213730f25bf64365c25ace13d4d464e7f31b26e556191d2f68f6498f077a6418c442dda81947d778b66c6/?fname=2f37356363313361322d336433642d346561612d383731302d396434383039663433616162'
  }
}`

直接根据这个url就可以下载了
